packages/models-library/src/models_library/services.py

```diff
@@ -167,16 +178,39 @@ class ServiceProperty(BaseModel):
         None, alias="defaultValue", examples=["Dog", True]
     )
 
+    # TODO: use discriminators
+    unit: Optional[str] = Field(
+        None, description="Units, when it refers to a physical quantity"
```

Comment:
ok so this guy is None when there is no units? I think this will not create a anyOf{"string", "null"} does it?

Reply:
No, as [discussed in PR #2154](https://github.com/ITISFoundation/osparc-simcore/pull/2154#discussion_r577596358)